sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ChangeFeedTest.java

```diff
@@ -546,4 +548,10 @@ private static void waitAtleastASecond(Instant befTime) throws InterruptedExcept
             Thread.sleep(100);
         }
     }
+
+    @Retention(java.lang.annotation.RetentionPolicy.RUNTIME)
```

Comment:
why are these needed in this PR?

are these features removed from testng? and we are introducing our own?

Reply:
yes. originally we are using Tag from here: https://javadoc.io/doc/org.testng/testng/7.1.0/org/testng/xml/dom/package-summary.html

but from 7.3.0, I can not find this package any more, so created a new one

Reply:
I see. are we currently using that testng tag for grouping tests or running a subset of tests?

how are we planning to wire those test groups against the new tag. any work needed?

Reply:
Currently Tag is only used in the changeFeedTests. It is not used as group tests etc. No other work is needed.

Currently I think  it mainly used as a custom attributes, which is used to differentiate which document is created: 

@BeforeMethod(groups = { "simple", "emulator" }, timeOut = SETUP_TIMEOUT)
    public void populateDocuments(Method method) {

        checkNotNull(method, "Argument method must not be null.");

        Tag tag = method.getDeclaredAnnotation(Tag.class);
        if (tag != null && "EnableFullFidelity".equalsIgnoreCase(tag.name())) {
            populateDocumentsInternal(method, true);
        } else {
            populateDocumentsInternal(method, false);
        }
    }